go-getting-started/main.go

```diff
@@ -54,112 +59,242 @@ func (j jokesHandler) parseSkipAndLimit(w http.ResponseWriter, r *http.Request)
 		w.WriteHeader(http.StatusBadRequest)
 		return Pagination{}, nil
 	}
-	if skip == limit {
-		w.WriteHeader(http.StatusBadRequest)
-	}
+
 	pagination := Pagination{Skip: skip, Limit: limit}
 	return pagination, nil
 }
 
-func (p jokesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (j jokesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 
 	if r.Method == http.MethodPost {
 		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
 		return
 	}
 
-	if len(p.jokes) == 0 {
+	if len(j.jokes) == 0 {
 		w.WriteHeader(http.StatusNoContent)
-		json.NewEncoder(w).Encode(p.jokes)
+		err := json.NewEncoder(w).Encode(j.jokes)
```

Comment:
You don't have to return any body along with the status 204 - https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html